.gitignore

```diff
@@ -1,3 +1,5 @@
+
+int main()
```

Comment:
Why would a `.gitignore` have `int main()`?

Reply:
Well we should consider this as a spam PR given what it's suggesting...